test/test_MPI.jl

```diff
@@ -3,6 +3,7 @@ using EcoSISTEM.Units
 using Unitful, Unitful.DefaultSymbols
 using Distributions
 using MPI
+using JLD
 
 @testset "MPI" begin
```

Comment:
I think this is only running with one process?

Reply:
Yes it is - but maybe that's fine, if we want to ensure that codecov picks up some of it?